cmd/podman/root.go

```diff
@@ -457,6 +457,8 @@ func rootFlags(cmd *cobra.Command, podmanConfig *entities.PodmanConfig) {
 		pFlags.StringVar(&podmanConfig.Runroot, runrootFlagName, "", "Path to the 'run directory' where all state information is stored")
 		_ = cmd.RegisterFlagCompletionFunc(runrootFlagName, completion.AutocompleteDefault)
 
+		pFlags.BoolVar(&podmanConfig.TransientStore, "transient-store", false, "Enable transient container storage")
```

Comment:
it seems to override the setting in `storage.conf`.

Shouldn't `transient_store = true` in `storage.conf` always set the transient flag?

Reply:
We need to be able to pass this in the conmon cleanup commands to get the exact same setup as the initial podman command was run as, similar to the --rundir=... args, etc. That can't rely on the global setting.

Reply:
agreed it makes sense to have the CLI option, but when I create a new container and the global setting is set I'd expect it to affect the newly created container unless I specify `--transient=false`

Reply:
Ah. right!

Reply:
Ok, this was a bit busted before, but should work now.

However, if you specify transient_store=true in the config file that currently only affects the default for rootful containers. It seems that there is a bunch of special code that defines the default for rootless containers. This might make sense, but I'm not sure, opinions?

Reply:
I don't feel strongly about either way.  I think it's a fairly niche feature (for now) but it's worth pointing users to storage.conf.